docs/data/material/components/dialogs/dialogs.md

```diff
@@ -137,12 +137,14 @@ Follow the [Modal limitations section](/material-ui/react-modal/#limitations).
 
 ## Complementary projects
 
-### Material UI Confirm
+For more advanced use cases you might be able to take advantage of:
 
-![stars](https://img.shields.io/github/stars/jonatanklosko/material-ui-confirm)
+### material-ui-confirm
+
+![stars](https://img.shields.io/github/stars/jonatanklosko/material-ui-confirm?style=social&label=Star)
 ![npm downloads](https://img.shields.io/npm/dm/material-ui-confirm.svg)
 
-This package provides dialogs for confirming user actions without writing boilerplate code.
+The package [`material-ui-confirm`](https://github.com/jonatanklosko/material-ui-confirm/) provides dialogs for confirming user actions without writing boilerplate code.
```

Comment:
I have seen at least 3 people complain about this. It's hard to find the source. Proof:

- https://mui-org.slack.com/archives/C041SDSF32L/p1682203740409249
- https://mui-org.slack.com/archives/C041SDSF32L/p1681988333141769
- https://mui-org.slack.com/archives/C041SDSF32L/p1681232882180359

I had to step in, this is IMHO a good signal/noise ratio